docs/ga4gh-pedigree.rst

```diff
@@ -0,0 +1,92 @@
+.. _rsga4ghpedigree:
+
+##############
+GA4GH Pedigree
+##############
+
+
+The Phenopacket-schema includes an implementation of the `GA4GH Pedigree Standard<https://pedigree.readthedocs.io/en/latest/index.html>`_.
+GA4GH Pedigree defines a conceptual model of individuals and their relationships to allow other standards, such as
+Phenopackets to be able to create an implementation of this model using native concepts. For example, the Phenopacket
+schema representation of an individual in the GA4GH Pedigree is a ref:`rstphenopacket`.
+
+The GA4GH Pedigree Standard allows for much greater expressivity of the relationships between individuals compared to the
```

Comment:
allows for -> enables (allows for is taking over but was wrong when I was in school :-0).

Reply:
I like it - it's more concise.